api/patient.py

```diff
@@ -1,5 +1,5 @@
 """
-@api {post} /patient/:user_id/create Create patient
+@api {post} /patient/create Create patient
```

Comment:
Factor out `patient` class by itself. `/patient/:user_id` does not quite make sense...